ios/Classes/MapboxMapController.swift

```diff
@@ -71,53 +72,59 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
             result(FlutterMethodNotImplemented)
         }
     }
-    
+
     private func updateMyLocationEnabled() {
-        //TODO
+      mapView.showsUserLocation = self.myLocationEnabled
+      mapView.userTrackingMode = .followWithHeading
```

Comment:
remove this line

Reply:
Same as @andrea689. The user tracking mode is set by a different method.